public/doc/openapi-3-v1.0.0.json

```diff
@@ -331,7 +331,7 @@
                   "Request"
                 ],
                 "summary": "Return an array of child requests of a given request id",
-                "description": "Return an array of child requests of a given request id, available for admin/requester",
+                "description": "Return an array of child requests of a given request id. The result depends on the x-rh-persona header (approval/admin, approval/requseter, or approval/approver).",
                 "operationId": "listRequestsByRequest",
                 "parameters": [
```

Comment:
We need to copy https://github.com/RedHatInsights/approval-api/blob/master/public/doc/openapi-3-v1.0.0.json#L207

Reply:
For `"responses"` part we need to copy https://github.com/RedHatInsights/approval-api/blob/master/public/doc/openapi-3-v1.0.0.json#L224